pipeline/core/core.py

```diff
@@ -28,7 +28,7 @@
 from astropy.modeling import (models, fitting, Model)
 from scipy import signal
 from ..info import __version__
-
+# __version__ = __import__('pipeline').__version__
```

Comment:
Is this a comment line? If it is, I recommend removing it. I believe we already have too many commented lines in the code and that we should avoid adding more.